packages/transition-frontend/ui-tests/login-tests.spec.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2025, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+
+import { test } from '@playwright/test';
+import * as testHelpers from './testHelpers';
+import * as loginTestHelpers from './loginTestHelpers';
+
+const context = {
+    page: null as any,
+    title: '',
+    widgetTestCounters: {}
+};
+
+// Configure the tests to run in serial mode (one after the other)
+test.describe.configure({ mode: 'serial' });
+
+test.beforeAll(async ({ browser }) => {
+    context.page = await testHelpers.initializeTestPage(browser);
+});
+
+// Login using all available methods
```

Comment:
COmment is a bit missleading, we have several other login method available in chaire-lib :)  
Maybe more details in the comment to say that it's for the method enabled in transition?